config/data/drug_stock_config.yml

```diff
@@ -1,4 +1,64 @@
-Punjab:
+Andhra Pradesh:
+  load_coefficient: 1
+  drug_categories:
+    hypertension_ccb:
+      new_patient_coefficient: 1.4
+      '329528': 1
+      '329526': 2
+      hypertension_arb:
+        new_patient_coefficient: 0.37
+        '316764': 1
+        '316765': 2
+      hypertension_diuretic:
+        new_patient_coefficient: 0.06
+        '429503': 1
+        '316049': 2
```

Comment:
Does the `1` and `2` here stand for doses per day? If this isn't captured anywhere already, it might be helpful to leave a comment with the expected structure of the YAML. `State > drug_category > rxnorm_code etc. ...`

Reply:
No, this is the `drug_coefficient` used for base doses calculations.